tests/unit/unit_t_db_base.py

```diff
@@ -132,7 +132,7 @@ def setUp(self):
         self.set_up_client()
 
     def set_up_client(self, auto_connect=False, auto_renew=False, encoder=None,
-                      timeout=300):
```

Comment:
I actually think it would be good if all our tests had a timeout. I just didn't think we needed to specify `300` if that was the default. Given that we're not applying a default until the next major we should probably apply one in all of our tests (except where it is obviously overridden to test the behaviour of the timeouts). Maybe `(30, 300)`?